src/app/public/modules/popover/popover.component.scss

```diff
@@ -181,3 +181,13 @@ $popover-max-width: 276px;
     }
   }
 }
+
+.sky-popover-contain-overflow {
+  .sky-popover {
+    overflow-y: auto;
```

Comment:
also needs a min-height of ~35px so that at least one row is always visible

Reply:
Great point. Done!